data/src/main/java/com/android254/data/network/models/payloads/Feedback.kt

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2022 DroidconKE
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.android254.data.network.models.payloads
+
+import kotlinx.serialization.KSerializer
+import kotlinx.serialization.Serializable
+import kotlinx.serialization.descriptors.PrimitiveKind
+import kotlinx.serialization.descriptors.PrimitiveSerialDescriptor
+import kotlinx.serialization.descriptors.SerialDescriptor
+import kotlinx.serialization.encoding.Decoder
+import kotlinx.serialization.encoding.Encoder
+
+@Serializable
+data class Feedback(val rating: FeedbackRating, val message: String)
+
+@Serializable(with = FeedbackRatingSerializer::class)
+enum class FeedbackRating { BAD, OKAY, GOOD }
+
+//
```

Comment:
Cleanup this?

Reply:
I used this to separate the model from the serializer for better readability. But it's just personal style; I'll remove it.

Reply:
For separation, regions would be much better